episodes/example.Rmd

```diff
@@ -112,6 +112,38 @@ accessibility purposes'}`
 
 ![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
 
+Additional attributes can be specified for the image alongside the alternative text description
+in the `{}`.
+Some, like `width` and `height`, can be specified directly:
+
+`![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.' width='25%'}`
+
+![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.' width='25%'}
+
+The Mozilla Developer Network provides [a list of the attributes that can be specified in the `{}` like this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attributes).
+
+:::::::::::: callout
+
+## :warning: Advanced Image Styling
```

Comment:
I think :art: would be better than a warning symbol.

```suggestion
## :art: Advanced Image Styling
```

Reply:
I chose warning because of the hazards of switching from `width=...` to `style='width:...'`. It certainly tripped me up! But perhaps the use of "Advanced" in the callout title serves a similar purpose. I will accespt your suggestions.